### PR TITLE
[EvalServer] AddModel.py to add models to models_for_eval

### PR DESCRIPTION
**Pull request by sethtroisi**
I'd like a review before I add the new cloud bigtable ('model_for_eval').

Then I'll test a couple of times before I send for final review

### CODE REVIEW

File: cluster/eval_server/add_model.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+flags.mark_flags_as_required([
+    "cbt_project", "cbt_instance", "cbt_table"
+])
+
+FLAGS = flags.FLAGS
+
+
+# TODO(sethtroisi): Unify with oneoffs
+MODEL_ROW = "m_{}_{:0>10}"
```
Comment (amj) [inline cluster/eval_server/add_model.py]: nit: give example in comment.

File: cluster/eval_server/add_model.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    assert "-num_readouts=" in flags, "must set playouts"
+    assert "-model" not in flags, "leave model unset"
+
+    # Verify binary_path is to a gtp binary and it exists
+    binary_base = binary_path.replace(GTP_BIN, "")
+    assert binary_path.endswith(GTP_BIN), binary_path
+    assert os.path.isfile(binary_path)
+    # $ ldd bazel-bin/cc/gtp
+    #    libtensorflow_framework.so => (bazel path) => cc/tensorflow/lib/libtensorflow_framework.so
+	#    libtensorflow_cc.so => (bazel path) => cc/tensorflow/lib/libtensorflow_cc.so
```
Comment (amj) [inline cluster/eval_server/add_model.py]: still needed?

File: cluster/eval_server/add_model.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    assert gfile.Exists(src)
+    assert not gfile.Exists(dst)
+
+    with gfile.GFile(src, "rb") as src_f, gfile.GFile(dst, "wb") as dst_f:
+        shutil.copyfileobj(src_f, dst_f)
+
+
+def add_model(name, model_path, flags, binary_path):
+    """Add a player (model + flags) to models cbt
+    name: name of the model
```
Comment (amj) [inline cluster/eval_server/add_model.py]: what is the name here, actually?  e.g. "000990-cormorant", or v17-990, or some other scheme?  This is a good place to take a no-op comment ("name: the name") and give some clarity.

File: cluster/eval_server/add_model.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+    # Upload model and binary
+    gcs_player_folder = os.path.join(PLAYER_FOLDER, player_hash)
+    new_model_path = os.path.join(gcs_player_folder, os.path.basename(model_path))
+    new_binary_base = os.path.join(gcs_player_folder, "bin")
+    binary_base = binary_path.replace(GTP_BIN, "")
+
+    print (f"Saving to {gcs_player_folder!r}")
+
+    #copy_to_gcs(model_path, new_model_path)
```
Comment (amj) [inline cluster/eval_server/add_model.py]: The print statement above is a lie?

(i'm assuming this is pending some clever way to dedupe the tf libs so as not to reup the same ones over and over?)

File: cluster/eval_server/add_model.py [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+    # validate that all rows written successfully
+    for status in enumerate(response):
+        if status.code is not 0:
+            print("Failed to write {}".format(i, status))
+
+
+if __name__ == '__main__':
+    remaining_argv = flags.FLAGS(sys.argv, known_only=True)
+    fire.Fire({
```
Comment (amj) [inline cluster/eval_server/add_model.py]: absl.app?  I'd like to avoid new uses of fire if possible.

File: cluster/eval_server/add_model.py (hunk 2 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: I think this illustratives why I'm copying these two files.

File: cluster/eval_server/add_model.py (hunk 3 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: it's a free form name field. I'll add an assert it conforms to [a-Z0-9-._]

Comment (amj) [reply cluster/eval_server/add_model.py]: ok.  I'm saying this is the chance to describe how the caller should use it.  I.e. how you think the names should be used or what makes a good name or an example...

File: cluster/eval_server/add_model.py (hunk 1 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: This get's automatically generated as m_eval_{HASH} see line ~104

File: cluster/eval_server/add_model.py (hunk 5 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: Let's discuss this offline but now passing args is harder :/

File: cluster/eval_server/add_model.py (hunk 4 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: fixed.

File: cluster/eval_server/add_model.py (hunk 5 above)
Comment (sethtroisi) [reply cluster/eval_server/add_model.py]: Fixed.

**Pull request merged**